dendronhq/dendron-11ty
# Feature/adding search build

Addresses #3 by building out the search index on 11ty start and copying the minified lunr library into place.

To test:
- Start up 11ty
- Confirm that the search-data.js file matches data requirements

Note:
I can't search yet, presumably because lunr isn't being initialised. I was unsure if this was being covered in this issue or another.
